kernel/nativecode.ml

```diff
@@ -67,6 +72,20 @@ let get_const_prefix env c =
    | NotLinked -> ""
    | Linked s -> s
 
+let get_const_lazy env =
+  let cache = ref Cmap_env.empty in
```

Comment:
Is the cache actually useful? This doesn't seem like a complex computation.

Reply:
It cannot hurt, at least, the amount of added code is tiny.